### ISSUE TITLE
Add Inception Score(IS) to metrax

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#82]

**Comment by Mrigankkh** [comment]
Hello, can I take this up?

**Comment by jshin1394** [comment]
of course :) We are trying to match the [torchmetrics implementation](https://lightning.ai/docs/torchmetrics/stable/image/inception_score.html) and follow other metrax metric API. Please feel free to reach out to me with specific questions or refer to other PRs for this repo!

**Comment by Mrigankkh** [comment]
Thank you for the torchmetrics implementation; super helpful. I'm on it!

**Comment by Mrigankkh** [comment]
Since IS depends on a pretrained InceptionV3 model , is there a preferred approach for handling this kind of external model dependency? Do you recommend using any existing JAX-compatible InceptionV3 implementation ?

For reference, the implementation of IS in torchmetrics uses FeatureExtractorInceptionV3 from torch-fidelity API.
[https://github.com/Lightning-AI/torchmetrics/blob/master/src/torchmetrics/image/inception.py]

Thanks in advance!

**Comment by jshin1394** [comment]
Hi Mrigank, thank you co much for your interest in our repo and this metric. Actually, given the complexity of this metric and how it will require some framework level modification due to its requirements for a pre-trained InceptionV3 model, I would like to reprioritize this metric. Would you be open to working on some of the other metrics from the repo issue? :)

**Comment by Mrigankkh** [comment]
Agreed! And yes I would love to pick up a different metric. Thank you! 

Also FYI , #62  [Add Frechet Inception Distance (FID) metric] also requires a pre-trained InceptionV3 model so I'd suggest checking it out to see if it needs to be reprioritized as well! :)

**Comment by jshin1394** [comment]
Thank you so much. I will also deprioritize FID as well.
